### PR TITLE
Misplaced Appearance type

### PR DESCRIPTION
**Pull request by cmarin001**
This pull request addresses an issue where the appearance type was incorrectly applied or misplaced within several UI components of our application. The goal is to correct this misplacement to ensure that UI components adhere to the intended design specifications and provide a consistent user experience.

### CODE REVIEW

File: src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx [hunk 1]
```diff
@@ -1,30 +1,39 @@
-import { useContext, useState } from "react";
+import { useContext, useEffect, useState } from "react";
 import { IUsersMessage } from "@pages/privileges/outlets/users/types/users.types";
 import { inube } from "@inube/design-system";
 import { paletteMessagesConfig } from "../../config/palette.config";
 import { RenderContentFormPaletteUI } from "./interface";
 import { TokenContext } from "@context/TokenContext";
 import { LoadingAppUI } from "@src/pages/login/outlets/LoadingApp/interface";
+import { PaletteAppearance } from "../../types";
```
Comment (LinconOssa) [inline src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx:8]: Would it be better to put @pages/people/outlets/color/palette/types?

File: src/pages/people/outlets/color/palette/form/RenderContentFormPalette/interface.tsx [hunk 2]
```diff
@@ -3,14 +3,14 @@ import { inube, Text, useMediaQueries } from "@inube/design-system";
 
 import { IMessageState } from "@pages/privileges/outlets/users/types/forms.types";
 import { ThemeProvider } from "styled-components";
-import { Appearance } from "@components/feedback/SendingInformation/types";
 import { RenderCategoryGrid } from "@components/layout/RenderCategoryGrid";
 import { paletteConfig } from "../../config/palette.config";
 import { RenderMessage } from "@components/feedback/RenderMessage";
+import { PaletteAppearance } from "../../types";
```
Comment (LinconOssa) [inline src/pages/people/outlets/color/palette/form/RenderContentFormPalette/interface.tsx:9]: Maybe in all the similar ones, would it be better to put the shape I gave it?

**Review by LinconOssa — COMMENTED** [review]
Just look if it is the best way to put ../../ or with @

File: src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx (hunk 1 above)
Comment (cmarin001) [reply src/pages/people/outlets/color/palette/form/RenderContentFormPalette/index.tsx]: solved

File: src/pages/people/outlets/color/palette/form/RenderContentFormPalette/interface.tsx (hunk 2 above)
Comment (cmarin001) [reply src/pages/people/outlets/color/palette/form/RenderContentFormPalette/interface.tsx]: solved

**Review by LinconOssa — APPROVED** [review]
meet the requirements

**Pull request merged**